websites/B/bilibilitv/metadata.json

```diff
@@ -8,6 +8,10 @@
 		{
 			"name": "Faelayis",
 			"id": "328731868096888833"
+		},
+		{
+			"name": "elaxan",
+			"id": "506212044152897546"
```

Comment:
In my opinion this isn't enough code change to become a contrbutor.

But it's up to @Bas950 & @theusaf